inbm/configuration-agent/configuration/broker.py

```diff
@@ -240,12 +241,13 @@ def _publish_agent_values(self, agent) -> None:
 
     def _publish_new_values(self, paths: str) -> None:
         path_list = paths.split(';')
+        logger.debug('publish new values')
         for i in range(0, len(path_list) - 1):
             list_obj = path_list[i].split(':', 1)
             value = list_obj[1]
             path = list_obj[0]
             self.mqttc.publish(
-                UPDATE_CHANNEL + str(path), json.dumps(value))
+                UPDATE_CHANNEL + str(path), json.dumps(value), retain=True)
```

Comment:
log statement should go here instead showing UPDATE_CHANNEL + str(path) and json.dumps(value), so e.g. we might log 'Publishing new value on /update/channel/path: NEW_VALUE'